native-android/src/lib/notifications.ts

```diff
@@ -60,15 +73,6 @@ export const initNotifications = async () => {
   });
   Notifications.registerTaskAsync(BACKGROUND_NOTIFICATION_TASK);
 
-  // Handle receiving notifications while app is in foreground
-  Notifications.setNotificationHandler({
```

Comment:
I think it's a good idea to take out foreground notifications, we should check with @jamesacklin too, though.